app/styles/app.scss

```diff
@@ -104,6 +104,11 @@ td .input-group {
     height: 30px;
   }
 }
+@media (min-width: 992px) {
```

Comment:
This works for most names, but in the case that there's a REALLY long name that takes up more space than the navbar, the text doesn't truncate like it does on screen sizes < 768px.  Would it be possible to add ellipses in the case that the string is longer than the screen?

I also noticed that this still truncates early between 768 - 991px.  It might be a good idea to keep those sizes consistent with the behavior of the larger/smaller sizes (since they all seem to be full width with the `...` at the end, even on smaller screens).